R/corrpart.b.R

```diff
@@ -246,72 +233,59 @@ corrPartClass <- R6::R6Class(
 
             return(data)
         },
-        .test = function(var1, var2, controls, type) {
+        .test = function(var1, var2, controls, type, hyp) {
 
             data <- private$.cleanData(var1, var2)
             var1 <- data[[var1]]
             var2 <- data[[var2]]
 
-            if (type != 'zero' && length(controls) > 0)
-                controls <- data[, controls]
-
-            results <- list()
+            n <- length(data[[1]])
+            d <- 1 + as.integer(type == 'semi')
+            c <- length(controls)
```

Comment:
Try using variable names that are a bit more verbose, for instance `nControls` instead of `c` (not sure what `d` is here).

Reply:
Resolved, changed n → nSubj, d → nResR (number of rows of results, 2 for semipartial, 1 otherwise), c → nCtrV (number of control variables)